etc/helm/pachyderm/templates/pgbouncer/deployment.yaml

```diff
@@ -55,14 +57,26 @@ spec:
           value: "300"
         - name: PGBOUNCER_IGNORE_STARTUP_PARAMETERS
           value: "extra_float_digits"
+        - name: PGBOUNCER_SERVER_TLS_SSLMODE
+          value: "{{ .Values.global.postgresql.postgresqlSSL }}"
+         {{- if .Values.global.postgresql.postgresqlSSLSecret  }}
+        - name: PGBOUNCER_SERVER_TLS_CERT_FILE
+          value: /pg-tls-cert/tls.crt
+        - name: PGBOUNCER_SERVER_TLS_KEY_FILE
+          value: /pg-tls-cert/tls.key
+         {{- end }}
+         {{- if .Values.global.postgresql.postgresqlSSLCACert  }}
+        - name: PGBOUNCER_SERVER_TLS_CA_FILE
+          value: /pg-ca-cert/ca.crt
+         {{- end }}
         - name: PGBOUNCER_UNIX_SOCKET_DIR
           value: " "
         - name: POSTGRESQL_PASSWORD
           valueFrom:
             secretKeyRef:
               name: {{ .Values.global.postgresql.postgresqlExistingSecretName | default "postgres" }} 
               key: {{ .Values.global.postgresql.postgresqlExistingSecretKey | default "postgresql-password" }}
-        image: bitnami/pgbouncer:1.16.0
+        image: bitnami/pgbouncer:1.16.1-debian-10-r82
```

Comment:
Do they really provide Debian 10 images?  Debian 11 is like a year old at this point :)